src/styles/mixins/_breakpoints.scss

```diff
@@ -0,0 +1,17 @@
+@mixin tablet {
+  @media (max-width: 1000px) {
+    @content;
+  }
+}
+
+@mixin large-mobile {
+  @media (max-width: 715px) {
+    @content;
+  }
+}
+
+@mixin mobile {
+  @media (max-width: 600px) {
+    @content;
+  }
+}
```

Comment:
The naming is a little open here, something like ex. `bp-tablet` would more clearly tell what it's doing. But that's just preference.